examples/stm32f469_discovery/usb/main.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2011, Fabian Greif
+ * Copyright (c) 2013, Kevin Läufer
+ * Copyright (c) 2013-2017, Niklas Hauser
+ * Copyright (c) 2014, Sascha Schade
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+#include <modm/processing.hpp>
+#include <tusb.h>
+#include <modm/io.hpp>
+
+using namespace Board;
+using UsbIoDevice = modm::IODeviceWrapper<UsbUart<0>, modm::IOBuffer::DiscardIfFull>;
+
+// ----------------------------------------------------------------------------
+int
+main()
+{
+	Board::initialize();
+
+	Board::usb::VBus::setInput();
+	Board::usb::Dp::Dp<Peripheral::Usbotgfs>::connect();
+	Board::usb::Dm::Dm<Peripheral::Usbotgfs>::connect();
+	Board::usb::Id::Id<Peripheral::Usbotgfs>::connect();
+
+	Rcc::enable<Peripheral::Usbotgfs>();
+	USB_OTG_FS->GCCFG |= USB_OTG_GCCFG_VBDEN;
```

Comment:
This should go into the UsbUart class IMO (e.g. `intialize` / `connect` functions)

Reply:
Agreed, just didn't get there yet.